qa/cephfs/begin.yaml

```diff
@@ -47,3 +47,4 @@ tasks:
         - libuuid-devel
         - xfsprogs-devel
   - ceph:
+      session_timeout: 300
```

Comment:
Let's make this an override so it's easier to apply only to a small number of tests that we expect to run into this problem. See

https://github.com/ceph/ceph/blob/473d8e9a6066edace841e7580245f624d0afaab1/qa/tasks/ceph.py#L1792

on how to access the overrides dictionary. You would do that from filesystem.py.

Reply:
Sure, will do it.